### PR TITLE
Remove delta vchannel when clean up empty flowgraph

### PR DESCRIPTION
**Pull request by congqixia**
Related to #20151

/kind bug

Signed-off-by: Congqi Xia <congqi.xia@zilliz.com>

### CODE REVIEW

File: internal/querynode/meta_replica.go [hunk 1]
```diff
@@ -853,6 +855,20 @@ func (replica *metaReplica) getSealedSegments() []*Segment {
...
+	replica.mu.RLock()
+	defer replica.mu.RUnlock()
+
+	coll, ok := replica.collections[collectionID]
+	if !ok {
+		// if collection already release, that's fine and just return
+		return
+	}
+
+	coll.removeVDeltaChannel(vDeltaChannel)
```
Comment (yah01) [inline internal/querynode/meta_replica.go:869]: lock collection here?

Comment (congqixia) [reply internal/querynode/meta_replica.go:869]: collection has `channelMut` for channel modification, which is used in the `removeVDeltaChannel`

**Pull request merged**